joomla/joomla-cms
# [4.0] [com_contact] Missing parameterized queries, fixed double quoting

### Summary of Changes

Adds some missing parameterized queries and fixes possible double quoting.

### Testing Instructions

On multilingual site create some contacts with associations.
View contact list, check that associations appear like before.
View feature view in frontend.
Changes in `components/com_contact/Service/Route.php` can be tested by code review. Not sure if it's actually used anywhere.

### Expected result

Works.

### Documentation Changes Required

No.
